core/modules/date/tests/date_field.test

```diff
@@ -417,17 +422,23 @@ class DateFieldTokenTestCase extends DateFieldBasic {
     foreach ($tokens as $name => $expected) {
       $token = $input[$name];
       if (!isset($expected)) {
-        $this->assertTrue(!isset($values[$token]), t("Token value for @token was not generated.", array('@token' => $token)));
+        $this->assertTrue(!isset($values[$token]), t("Token value for @token was not generated.", array(
+          '@token' => $token,
+        )));
       } else {
-        $this->assertIdentical($replacements[$token], $expected, t("Token value for @token was '@actual', expected value '@expected'.", array('@token' => $token, '@actual' => $replacements[$token], '@expected' => $expected)));
+        $this->assertIdentical($replacements[$token], $expected, t("Token value for @token was '@actual', expected value '@expected'.", array(
+          '@token' => $token,
+          '@actual' => $replacements[$token],
+          '@expected' => $expected,
+        )));
```

Comment:
Since we are touching these...
- the `$message` variable in assertions should NOT be passed through `t()` - `format_string()` can be used instead if there is a need for placeholder/replacements. See https://github.com/backdrop/backdrop-issues/issues/6277
- also use single quotes instead of double quotes where possible.

Reply:
The reason for the double quotes is that there is literal quotes within the text. That can be escaped, I guess.

Reply:
Resolved in latest commit.